### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by boostedgti916** [taoteh1221/Slideshow_Crypto_Ticker#33] [General]
Will running the regular cmds apt update and upgrade update this package?

**Comment by taoteh1221** [comment]
No, you'll need to re-run the same auto-installer you installed with (it also upgrades the app). See the "AUTOMATIC INSTALLATION" section in README.txt: https://raw.githubusercontent.com/taoteh1221/Slideshow_Crypto_Ticker/main/README.txt
